.github/workflows/ci.yaml

```diff
@@ -205,6 +205,15 @@ jobs:
           name: "npm-package"
           path: release-npm-package
 
+      # NOTE@jsjoeio - we need to make sure we're using
+      # v7 or higher of the npm CLI because it's used
+      # in the yarn publish:npm script in development builds
+      # to modify package.json name.
+      - name: Install Node & npm v8
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.14.0"
```

Comment:
We don't need this specific version, but I think it's better to use a specific version so we it's consistent and we don't get any surprises from it changing (if we were to only specify "16")

Reply:
We also only need this in development, which is why I only changed it in `ci.yaml` and not the other places where we use `yarn publish:npm`

Reply:
agreed! makes sense

Reply:
@jawnsy P.S. this is something I _specifically_ learned from you! ♥️